PluginServertastic.php

```diff
@@ -4,236 +4,474 @@
 
 class PluginServertastic extends SSLPlugin
 {
-    public $mappedTypeIds = array (
+    public $mappedTypeIds = array(
         SSL_CERT_RAPIDSSL                               => 'RapidSSL',
         SSL_CERT_RAPIDSSL_WILDCARD                      => 'RapidSSLWildcard',
+        SSL_CERT_COMODO_POSITIVESSL                     => 'PositiveSSL',
+        SSL_CERT_COMODO_POSITIVESSL_WILDCARD            => 'PositiveSSLWildcard',
         SSL_CERT_GEOTRUST_QUICKSSL_PREMIUM              => 'QuickSSLPremium',
         SSL_CERT_GEOTRUST_TRUE_BUSINESSID               => 'TrueBizID',
         SSL_CERT_GEOTRUST_TRUE_BUSINESSID_EV            => 'TrueBizIDEV',
         SSL_CERT_GEOTRUST_TRUE_BUSINESSID_WILDCARD      => 'TrueBizIDWildcard',
         SSL_CERT_VERISIGN_SECURE_SITE                   => 'SecureSite',
-        SSL_CERT_VERISIGN_SECURE_SITE_PRO               => 'SecureSitePro',
         SSL_CERT_VERISIGN_SECURE_SITE_EV                => 'SecureSiteEV',
+        SSL_CERT_VERISIGN_SECURE_SITE_PRO               => 'SecureSitePro',
         SSL_CERT_VERISIGN_SECURE_SITE_PRO_EV            => 'SecureSiteProEV',
```

Comment:
In one of my commits I stated that I renamed these constants to Symantec but I changed my mind as I didn't want to break other SSL plugins relying on these constants.